app/editor/src/features/content/form/PropertiesContentForm.tsx

```diff
@@ -2,46 +2,94 @@ import { Button, ButtonVariant } from 'components/button';
 import { Col } from 'components/col';
 import { IOptionItem, OptionItem, RadioGroup, SelectDate } from 'components/form';
 import { FormikCheckbox, FormikDropdown, FormikText, FormikTextArea } from 'components/formik';
+import { Modal } from 'components/modal/Modal';
 import { Row } from 'components/row';
+import { Upload } from 'components/upload';
 import { useFormikContext } from 'formik';
+import { ITagModel, ITimeTrackingModel, IUserModel } from 'hooks';
+import useModal from 'hooks/modal/useModal';
 import React from 'react';
 import { useLookup } from 'store/hooks';
+import { useKeycloakWrapper } from 'tno-core';
+import { getSortableOptions } from 'utils';
 
 import { expireOptions, summaryOptions, toningOptions } from './constants';
 import { IContentForm } from './interfaces';
+import { TimeLogTable } from './TimeLogTable';
 
 export interface IContentSubForms {
   setContent: (content: IContentForm) => void;
   content: IContentForm;
 }
 
+/** The sub form of the ContentForm when the Properties Tab is selected. */
 export const PropertiesContentForm: React.FC<IContentSubForms> = ({ setContent, content }) => {
-  const [{ series: lSeries }] = useLookup();
-  const { values } = useFormikContext<IContentForm>();
+  const [{ series: lSeries, categories, tags, users }] = useLookup();
+  const { values, setFieldValue, handleChange } = useFormikContext<IContentForm>();
+  const [userTags, setUserTags] = React.useState<string[]>();
+  const [categoryTypes, setCategoryTypes] = React.useState<IOptionItem[]>([]);
+  const keycloak = useKeycloakWrapper();
+  const userId = users.find((u: IUserModel) => u.displayName === keycloak.getDisplayName())?.id;
```

Comment:
Display name is not good to bind to.  It is supposed to be a friendly name a user changes for display purposes.  Use their unique username or uid.

Reply:
Will do!